packages/sitecore-jss-react-native/.eslintrc

```
@@ -1,19 +1,12 @@
 {
-  "extends": ["../../.eslintrc"],
-  "parser": "@typescript-eslint/parser",
-  "plugins": ["react-native", "@typescript-eslint"],
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
-    }
-  },
+  "extends": [
+    "../../.eslintrc",
+    "../../eslint-configs/typescript"
+  ],
+  "plugins": [
+    "react-native",
```

Remove trailing comma
```suggestion
    "react-native"
```